### DISCUSSION TITLE
Help with search string / multiple selections

### DISCUSSION THREAD

**Discussion by Blinglebob** [deckerst/aves#1360] [Q&A] [answered]
Hi, I have recently discovered Aves and it seems great! I have been adding tags and am now wondering how I can  search/filter them. 
I found the Search on collection can display images with a single tag, but I cannot seem to find a way to add multiple tags to the search, is this possible?
Also is it possible to construct any search query here? Eg.  A search for tag="wife","dog" to return all images tagged with either wife or dog. Or tag="wife"&"dog" to return pictures where only images containing both tags are returned?
Maybe this exists already but I have discovered where to find it yet.
Looking forward to your reply.
Thank you.

**Comment by deckerst** [comment, marked as answer]
Fire "and", you can tap on any filter in the Search page and it will add it to the current collection. You can also add filters via the Stats page, and the Info page of an item. 

For "or", the only way is to select multiple items on the Tags page, then in the top right menu select "Show in Collection".

**Reply by Blinglebob** [reply]
Perfect! Thank you. That is exactly what I was looking for. It's so simple now I know it.
